test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("Horizontal pod autoscaling (scale resource: CPU) [Skipped]", f
 	titleUp := "Should scale from 1 pod to 3 pods and from 3 to 5"
 	titleDown := "Should scale from 5 pods to 3 pods and from 3 to 1"
 
-	Describe("Deployment", func() {
+	Describe("Deployment [Feature:Deployment]", func() {
```

Comment:
@piosz It looks like you're not running these tests in your autoscaling suite right now; can we re-enable them?